### PR TITLE
Properly detect conflicts on windows

### PR DESCRIPTION
**Pull request by romanowski**
[test_windows_full]

### CODE REVIEW

**Comment by romanowski** [comment]
fixes #13176

File: scaladoc/src/dotty/tools/scaladoc/renderers/Resources.scala [hunk 1]
```diff
@@ -171,8 +171,9 @@ trait Resources(using ctx: DocContext) extends Locations, Writer:
   )
 
   def renderResource(resource: Resource): Seq[String] =
-    if resource.path.endsWith(".html") && apiPaths.contains(resource.path) then
-      report.error(s"Conflict between resource and API member for ${resource.path}. $pathsConflictResoultionMsg")
+    val normalizedPath = resource.path.replace('\\', '/')
```
Comment (BarkingBad) [inline scaladoc/src/dotty/tools/scaladoc/renderers/Resources.scala:174]: Wouldn't be `Paths.get(resource.path).toString` more elegant in that case?

**Review by BarkingBad — APPROVED** [review]
Looks OK, just a small nitpick

**Comment by bishabosha** [comment]
can we merge this?

File: scaladoc/src/dotty/tools/scaladoc/renderers/Resources.scala (hunk 1 above)
Comment (romanowski) [reply scaladoc/src/dotty/tools/scaladoc/renderers/Resources.scala:174]: It may still return representation with `\\`

**Comment by bishabosha** [comment]
> I am not sure why CI restarted for some tests, but I think once everything is green we can merge it.

sorry, I restarted it because the ci had failed previously due to the `scaladoc / stdlib-sourcelinks-test`

**Pull request merged**